src/app/components/Contact.tsx

```
@@ -0,0 +1,174 @@
+                      WhatsApp +62 812-8895-7868
+                    </h4>
+                    <p className="text-gray-600 text-sm">
+                      Chat langsung untuk order cepat
+                    </p>
+                  </div>
+                </a>
+
+                <a
+                  href="https://www.instagram.com/mata.screenprint"
+                  target="_blank"
+                  className="flex items-center gap-4 p-4 bg-pink-50 rounded-lg hover:bg-pink-100 transition-colors"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)